Sprint-3/revise/implement/password-validator.js

```diff
@@ -0,0 +1,44 @@
+let passwords = [];
+
+function passwordValidator(input) {
+
+    // have to go learn about regex or validation methods for this exercise
+    //Cant use this: /[a-z]{1,}[A-Z]{1,}[0-9]{1,}[!#$%.*&]{1,}/
+    //The above is for a password that goes in that specific order
+
+    //const regexPattern = /[a-z]{1,}[A-Z]{1,}[0-9]{1,}[!#$%.*&]{1,}/;
+    //write a regex for each condition
+    const lowercaseRegex = /^(?=.*[a-z])/;
```

Comment:
Why not just specify the regular expression as `/[a-z]/` instead of `/^(?=.*[a-z])/`?

Reply:
I tried specifying it that way as `/[a-z]/ ` but it only worked when certain characters were in a specific order, it could not be randomized and looking at the password validator I don't belive that the passwords requires certain characters to come after each other

Reply:
What string value was you testing when the test gives you a different result between
`/[a-z]/.test(str)` and `/^(?=.*[a-z])/.test(str)` (where `str` holds a string value)?

You can also try using ChatGPT to help you understand what these expressions mean and find out their difference.

Reply:
I am not sure how you will get notified if I just leave my comments here. So I will also tag you  @sernamalala.

Reply:
Hi. Yes I see what you mean, my understanding of the regex and how it works was off. There is a difference but I can use this `/[a-z]/.test(str)` expression for my code.